src/FwdState.cc

```diff
@@ -128,7 +128,8 @@ FwdState::FwdState(const Comm::ConnectionPointer &client, StoreEntry * e, HttpRe
     waitingForDispatched(false),
     destinations(new ResolvedPeers()),
     pconnRace(raceImpossible),
-    storedWholeReply_(nullptr)
+    storedWholeReply_(nullptr),
+    peeringTimer(((assert(r)), r->hier.totalResponseTime()))
```

Comment:
I checked that request is present here and in tunnel.cc. We may want to replace this cumbersome construction with a helper checking function.